prog/weaver/main.go

```diff
@@ -69,6 +71,7 @@ func main() {
 	flag.IntVar(&bufSzMB, "bufsz", 8, "capture buffer size in MB")
 	flag.StringVar(&httpAddr, "httpaddr", fmt.Sprintf(":%d", weave.HTTPPort), "address to bind HTTP interface to (disabled if blank, absolute path indicates unix domain socket)")
 	flag.StringVar(&iprangeCIDR, "iprange", "", "IP address range to allocate within, in CIDR notation")
+	flag.StringVar(&defltSubnet, "defaultsubnet", "", "default subnet for IP address allocation, in CIDR notation")
```

Comment:
That's a horrid name. It's not at all clear that this option pertains to ipam. arguably the same is true of `iprange`. `ipamuniverse` and `ipamnet`?

Descriptions: "IP address range reserved for automatic allocation." and "subnet of ipamuniverse to allocate within by default".